src/App.tsx

```diff
@@ -15,58 +16,67 @@ export const goods = [
   'Garlic',
 ];
 
-export const App: React.FC = () => (
-  <main className="section container">
-    <h1 className="title">No goods selected</h1>
+export const App: React.FC = () => {
+  const [good, SetGood] = useState('Jam');
 
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
-      {/* eslint-disable-next-line jsx-a11y/control-has-associated-label */}
-      <button data-cy="ClearButton" type="button" className="delete ml-3" />
-    </h1>
+  return (
+    <main className="section container">
+      {!good ? (
+        <h1 className="title">No goods selected</h1>
+      ) : (
+        <h1 className="title is-flex is-align-items-center">
+          {`${good} is selected`}
+          <button
+            data-cy="ClearButton"
+            type="button"
+            className="delete ml-3"
+            onClick={() => {
+              SetGood('');
+            }}
```

Comment:
Create separate methods instead of creating inline event handlers (if we do so - we don't create new arrow function after component re-rendering) [[checklist]](https://github.com/mate-academy/react_goods-selector/blob/master/checklist.md)